tests/unit/pages/subjects/Subjects.spec.jsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import Subjects from '~/pages/subjects/Subjects'
+import { ModalProvider } from '~/context/modal-context'
+
+
+vi.mock('~/context/modal-context', () => ({
+  useModalContext: () => ({
+    closeModal: vi.fn()
+  }),
+  ModalProvider: vi.fn(({ children }) => <div>{children}</div>)
+}))
+
+vi.mock('~/components/page-wrapper/PageWrapper', () => {
+  const PageWrapperMock = ({ children }) => <div>{children}</div>
+  return {
+    __esModule: true,
+    default: vi.fn().mockImplementation(PageWrapperMock)
+  }
+})
+
+describe('Subjects Component', () => {
+  test('renders "Subjects" text within the PageWrapper', () => {
```

Comment:
Use word it, not test in test('renders "Subjects" text within the PageWrapper', () => {

Reply:
Done, thanks
![image](https://github.com/Space2Study-UA-1125/frontEnd/assets/93076661/b4e80338-30e4-4bc1-8c65-4a843f5ed6b9)